.github/workflows/land-blocking.yml

```diff
@@ -32,7 +32,24 @@ jobs:
       - name: Build, tag and push images
         if: steps.check_ks.outputs.should_run == 'true'
         run: |
-          docker/build-aws.sh --build-all --version $LIBRA_GIT_REV --addl_tags canary,${TEST_TAG}
+          set +e
```

Comment:
Do we know if this only valid for this code block? (not for everything in this file after that?)
@bors-libra delegate+

Reply:
Yes, I believe each step runs in a separate shell process

![image](https://user-images.githubusercontent.com/796949/76569271-ee470280-646f-11ea-98a3-fda76f3bfa5e.png)